packages/ponder-subgraph/package.json

```diff
@@ -15,7 +15,7 @@
     "Subgraph"
   ],
   "exports": {
-    ".": "./src/index.ts"
+    ".": "./dist/index.js"
```

Comment:
this change should be reverted, as it defines the esm-specific import, which should be the index.ts file